src/navargus/glue.py

```diff
@@ -559,9 +568,28 @@ def get_api_timeout(self) -> float:
         """Returns the configured API request timeout value"""
         return float(self.get("api", {}).get("timeout", 2.0))
 
-    def get_sync_on_startup(self):
-        """Returns True if this program should always sync the Argus API on startup"""
-        return bool(self.get("api", {}).get("sync-on-startup"))
+    def get_sync_interval(self):
+        """
+        Returns the interval in minutes of how often the Argus API is synced if
+        it is None, no re-sync should happen
```

Comment:
Looks like two sentences were joined together with no period or capitalization.

Referring to [PEP-257](https://peps.python.org/pep-0257/), I suggest this:

```suggestion
        """Returns the configured sync interval in minutes.
        
        The sync interval determines how often a full re-sync against the Argus API
        should take place. A value of `None` means that periodic re-sync should not
        take place.

```